packages/dashboard/test/dashboard.test.ts

```diff
@@ -308,11 +392,30 @@ describe('dashboard', () => {
       it('should unhide remove button when editable', async () => {
         dashboard.editable = true;
         await nextFrame();
-        const widget = getElementFromCell(dashboard, 1, 0) as DashboardSection;
+        const widget = getElementFromCell(dashboard, 1, 0) as DashboardWidget;
         const section = widget.closest('vaadin-dashboard-section') as DashboardSection;
         const removeButton = getRemoveButton(section);
         expect(removeButton.getBoundingClientRect().height).to.be.above(0);
       });
+
+      describe('i18n', () => {
+        it('should localize section', async () => {
+          dashboard.i18n = {
+            ...dashboard.i18n,
+            selectSectionTitleForEditing: 'foo',
+          };
+
+          await nextFrame();
+
+          const widget = getElementFromCell(dashboard, 1, 0) as DashboardWidget;
+          const section = widget.closest('vaadin-dashboard-section') as DashboardSection;
+          expect(section.i18n.selectSectionTitleForEditing).to.equal('foo');
+          expect(section.i18n).to.eql({
```

Comment:
Nit: Can be simplified to `expect(section.i18n).to.eql(dashboard.i18n);`

Reply:
Done